configure.ac

```diff
@@ -483,6 +477,10 @@ AC_SUBST([PART_LDFLAGS])
 CFLAGS=$SAVE_CFLAGS
 LDFLAGS=$SAVE_LDFLAGS
 
+
+# For disk notification and fault LED.
+PKG_CHECK_MODULES([LIBLSM], [libstoragemgmt >= 1.3.0])
```

Comment:
The `have_lsm` is dedicate for storaged lsm module which require another dependency `libconfig`.